root_orchestrator/system-manager-python/system_manager.py

```diff
@@ -104,11 +104,13 @@ def handle_init_client(message):
 
     cluster_address = sanitize(request.remote_addr)
 
-    cid = mongo_upsert_cluster(cluster_ip=cluster_address, message=message)
-    x = {"id": str(cid)}
+    message["cluster_ip"] = cluster_address
+    cluster = cluster_operations.create_cluster(message)
+    cid = str(cluster["_id"])
+    x = {"id": cid}
```

Comment:
Could you maybe place this "x" content directly into the " emit("sc2", json.dumps(x), namespace="/register")" below?
Otherwise, this is just some random/cryptic var.
i.e. ~  emit("sc2", json.dumps({"id": cid}), namespace="/register")

Reply:
makes sense!